pkgs/development/php-packages/deployer/default.nix

```diff
@@ -1,29 +1,21 @@
-{ mkDerivation, fetchurl, makeWrapper, installShellFiles, lib, php }:
+{ lib, fetchFromGitHub, php }:
 
-mkDerivation rec {
+php.buildPhpProject (finalAttrs: {
   pname = "deployer";
-  version = "6.8.0";
+  version = "7.3.1";
 
-  src = fetchurl {
-    url = "https://deployer.org/releases/v${version}/${pname}.phar";
-    sha256 = "09mxwfa7yszsiljbkxpsd4sghqngl08cn18v4g1fbsxp3ib3kxi5";
+  src = fetchFromGitHub {
+    owner = "deployphp";
+    repo = "deployer";
+    rev = "d99377d71ebbc39a586b39486c5a92ef6d3b7074";
+    hash = "sha256-bsJcozAVvRXUNdxRdBh5SB6qESUSXkkPeEWF50IiboQ=";
   };
 
-  dontUnpack = true;
+  composerLock = ./composer.lock;
```

Comment:
Why not use the lock file in the repo? It only gets cleaned in the tagged commit.

Reply:
Fixed.